packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx

```diff
@@ -82,6 +84,7 @@ export const ConfigurationPage = (props: ConfigurationPageProps): JSX.Element =>
   const [audioState, setAudioState] = useState<PermissionState | 'unsupported' | undefined>(undefined);
   /* @conditional-compile-remove(call-readiness) */
   getDevicePermissionState(setVideoState, setAudioState);
+  const modalLayerHostId = useId('callReadinessModalLayerhost');
```

Comment:
We already have a composite-root level modal layer host id, can we use this instead so all modals, context menus etc. going forward all bind to the same root:
https://github.com/Azure/communication-ui-library/blob/d7d036e2d316fdc8081c3858b5a2a0bbf9db7b1b/packages/react-composites/src/composites/CallComposite/CallComposite.tsx#L377

Reply:
Good idea, I confirmed with FluentUi modal examples that multiple modals use 1 root modal layer host. 
I'll make the change now!

Reply:
Alright! Done, needed to play around with some transform in the modal but got it to work with the current layerhost. 
Also removed mobileView only for that specific layerhost as I found that it was not needed during my testing with the PIP modal. (but needs to be removed for call readiness modal to render on desktop using that layerhost).